Raven.Studio.Html5/App/commands/database/documents/getCollectionsCommand.ts

```diff
@@ -55,7 +56,12 @@ class getCollectionsCommand extends commandBase {
                     var collections = terms.map(term => new collection(term, this.ownerDb, 0));
                     new getCollectionsCountCommand(collections, this.ownerDb)
                         .execute()
-                        .done(result => task.resolve(result))
+                        .done(r => {
+                            new getCollectionsLabelsCommand(collections, this.ownerDb)
```

Comment:
Move this logic to shell::fetchStudioConfig. As a side node getCollectionsCountCommand and getCollectionsLabelsCommand are independed so you could use $.when(task1, task2) to parallelise those requests. Instead of chaining one after the other.